tests/utils/log/elasticmock/__init__.py

```diff
@@ -48,6 +66,7 @@ def _get_elasticmock(hosts=None, *args, **kwargs):
 
 
 def elasticmock(f):
```

Comment:
**f**ile or **f**unction ? That is what I meant with "being careful" with `good-names`.

We defined `f` as [shorthand for file](https://github.com/apache/airflow/blob/master/pylintrc#L454), but in this case it is used probably for function. So we should change this probably to:
```suggestion
 def elasticmock(function):
```
(or func if this works)

Reply:
Right. function is good.